internal/gouging/gouging.go

```diff
@@ -190,7 +188,11 @@ func checkPriceGougingPT(gs api.GougingSettings, cs api.ConsensusState, txnFee t
 	}
 
 	// check LatestRevisionCost - expect sane value
-	maxRevisionCost, overflow := gs.MaxRPCPrice.AddWithOverflow(gs.MaxDownloadPrice.Div64(bytesPerTB).Mul64(2048))
+	twoKiBMax, overflow := gs.MaxDownloadPrice.Mul64WithOverflow(2048)
+	if overflow {
+		twoKiBMax = types.MaxCurrency
+	}
+	maxRevisionCost, overflow := gs.MaxRPCPrice.AddWithOverflow(twoKiBMax)
```

Comment:
This looks off, we want to check whether `LatestRevisionCost` is sane and when we overflow we compare it to `types.MaxCurrency`, we should probably set it to zero?

Reply:
Setting `MaxDownloadPrice` and `MaxRPCPrice` to large values is sane right? We can't expect the user to account for the math we perform on those values. So if that ends up resulting in a very large number we just cap it at the max.